frontend/src/react-query/api/schema.tsx

```diff
@@ -0,0 +1,469 @@
+import { ConnectError } from '@connectrpc/connect';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
```

Comment:
Maybe explicitly state and add alias that this is tanstack, since we usually may not notice upon 1st sight that this is not Connect/RPC realm, so something like what we do for `frontend/src/react-query/api/topic.tsx`
```tsx
import {
  useQueryClient,
  useMutation as useTanstackMutation,
  useQuery as useTanstackQuery,
} from '@tanstack/react-query';
```